src/View/layout.html.twig

```diff
@@ -6,7 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{% block title %}{% endblock %}</title>
     <link rel="stylesheet" href="/assets/css/style.css">
-    <link rel="icon" href="/assets/images/favicon.png">
+    <link rel="icon" href="/assets/images/castle-50.png">
+    <link href="https://fonts.googleapis.com/css2?family=MedievalSharp&display=swap" rel="stylesheet">
```

Comment:
You could have your fonts in your repository instead of forcing your users to request fonts to google each time they visit your site.

Reply:
I wanted to test if the style of the font applies everywhere. It would obviously be more judicious to put the source code of the font directly in the "assets" folder of the game.